listenbrainz/webserver/static/js/src/APIService.ts

```diff
@@ -119,6 +119,30 @@ export default class APIService {
     return { status: response.status };
   };
 
+  getFollowers = async (username: string) => {
```

Comment:
The name here can be more expressive, I went with `getFollowersOfUser` in the backend code.